### PR TITLE
I made a pull request

### PR DESCRIPTION
**Pull request by AntoninDighera**
Making pull request

### CODE REVIEW

File: Dom.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+I made my whole code folder a repository, but I cloned more
+ repositories inside of that and I just never push it to anything.
```
Comment (zifn) [inline Dom.txt]: This is a comment

Comment (margaret) [inline Dom.txt]: 👍 😸

Comment (margaret) [reply Dom.txt]: wait 😿

Comment (doramedrano) [inline Dom.txt]: Good job! 😄

**Pull request merged**